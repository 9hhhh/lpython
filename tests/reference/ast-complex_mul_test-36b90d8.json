{
    "basename": "ast-complex_mul_test-36b90d8",
    "cmd": "lfortran --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/complex_mul_test.f90",
    "infile_hash": "908ba75c567d6ff4ba0e42cba3e974999d00e5d24710f4649ac212d6",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-complex_mul_test-36b90d8.stdout",
    "stdout_hash": "ee5f4dce8bedc988396874bf47560388b7c4e63c86be10e89747d888",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}