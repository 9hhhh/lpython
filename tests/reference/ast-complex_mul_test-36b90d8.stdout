(TranslationUnit [(Program complex2 [] [(Declaration [(x "complex" [] [] [] ())])] [(= x (Complex (Real "3.0") (Real "4.0"))) (= x (* x (Real "4.0"))) (Print () [x]) (= x (* 2 x)) (Print () [x]) (= x (* x (Complex (Real "0.0") (Real "3.0")))) (Print () [x])] [])])
